lib/pod/command/package.rb

```diff
@@ -67,10 +67,19 @@ def build_in_sandbox(platform)
 
         sandbox = install_pod(platform.name)
 
-        perform_build(platform, sandbox)
-
-        Pathname.new(config.sandbox_root).rmtree
-        FileUtils.rm_f('Podfile.lock')
+        begin
+          perform_build(platform, sandbox)
+
+        rescue Pod::Builder::BuildFailedException => ex
+          puts "Build command failed: #{ex.command}"
+          puts "Output:"
+          ex.output.each { |line| puts "    #{line}" }
+          Process.exit
```

Comment:
Coming back to this today, I agree. I'll move the code in this `rescue` block into `Builder`.